File: Lab7Calificado/Lab7_2026_1/PokeSoftSolution/PokeSoftModel/TipoPokemon.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PokeSoftModel
{
    public class TipoPokemon
    {
        private int _idTipoPokemon;
        private String? _nombre;
        public TipoPokemon() { }
        public TipoPokemon(String? nombre)
        {
            _nombre = nombre;
        }
        public TipoPokemon(int idTipoPokemon, String? nombre)
        {
            _idTipoPokemon = idTipoPokemon;
            _nombre = nombre;
        }
        public int IdTipoPokemon { get => _idTipoPokemon; set => _idTipoPokemon = value; }
        public String? Nombre { get => _nombre; set => _nombre = value; }
    }
}

File: lab11Calificado/Planchazo-Lab-11-Pokemon/PokemonsProg/PokemonesModel/src/main/java/pe/edu/pucp/pokemones/model/TipoPokemon.java
package pe.edu.pucp.pokemones.model;

public class TipoPokemon {
    private int id_Tipo_Pokemon;
    private String nombre;

    public TipoPokemon(){

    }


    //Getters and setters
    public int getIdTipoPokemon() {
        return id_Tipo_Pokemon;
    }

    public void setIdTipoPokemon(int id_Tipo_Pokemon) {
        this.id_Tipo_Pokemon = id_Tipo_Pokemon;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }


}
